structure/parser/object_parser.go

```diff
@@ -204,6 +204,8 @@ func (o *Object) Time(reference string, layout string) *time.Time {
 		return nil
 	}
 
+	// TODO: since old unit tests directly test feeding strings into this parser, removing this
```

Comment:
Please open a ticket if this must be fixed in the future

Reply:
Code sadly cannot be cleaned up without splitting the Request parser off to not share this code, unit tests were already added to ensure it works as intended, so this TODO was not actionable.